USE-CASES/hazard-annotations.md

```diff
@@ -0,0 +1,135 @@
+## Title: Connected Hazards
+
+### Submitter(s):
+
+Luca Barbato
+
+### Reviewer(s):
+
+<Suggest reviewers>
+
+### Tracker Issue ID:
+
+<please leave blank>
+
+### Category:
+
+"vertical"
+
+### Class:
+
+<please leave blank>
+
+### Status:
+
+<please leave blank>
+
+### Target Users
+
+- device owners
+- device users
+- device manufacturers
+
+
+### Motivation:
+
+Connected devices have an impact on the physical world, more than often interacting with them may have unintended consequences.
+
+The use-case originated within [SIFIS-Home](https://sifis-home.eu) project, where the focus is on trustworthy connected homes:
+The user should be made fully aware of the hazards related to the connected devices operations, be able to declare policies regarding acceptable risks, and have the home system enforce them.
+
+The focus is on the problem of having the information available and how to deliver it to all the possible actors.
+Few scenarios on how the information could be consumed in useful and advanced ways are provided.
+
+### Expected Devices:
+
+Any device that may pose a physical or privacy hazard to the user should provide information on the risks e.g.:
+    - Fire hazards: Ovens, Stoves, Lamps
+    - Flood hazards: Windows, Faucets
```

Comment:
I know this is an "e.g." list and so not meant to be exhaustive, but a few more hazards would be helpful (eventually these can become "hazard categories" in an ontology:
```suggestion
    - Flood hazards: Windows, Faucets
    - Chemical hazards: CO emissions e.g. for cars or heating systems, Battery venting
    - Electrical hazards: Solar panels, Batteries
```